server/src/main/java/au/org/aodn/ogcapi/server/processes/RestApi.java

```diff
@@ -0,0 +1,123 @@
+package au.org.aodn.ogcapi.server.processes;
+
+
+import au.org.aodn.ogcapi.processes.api.ProcessesApi;
+import au.org.aodn.ogcapi.processes.model.Execute;
+import au.org.aodn.ogcapi.processes.model.InlineResponse200;
+import au.org.aodn.ogcapi.processes.model.ProcessList;
+import au.org.aodn.ogcapi.server.core.model.enumeration.DatasetDownloadEnums;
+import au.org.aodn.ogcapi.server.core.service.AWSBatchService;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.enums.ParameterIn;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.Valid;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+@RestController("ProcessesRestApi")
+@RequestMapping(value = "/api/v1/ogc/processes")
+public class RestApi implements ProcessesApi {
+
+    @Autowired
+    private AWSBatchService awsBatchService;
+
+    @PostMapping(path = "/download/{collectionId}")
```

Comment:
You do not need to create this function, please use /processes/processID/execution to submit the job, later you should be able to use the JobApi to check the status and get result.

the processId is an id to the type of job, in this case may be "download" and then you can use the Execution body to identify what kind of download